WalletWasabi.Gui/Controls/StatusBar.xaml

```diff
@@ -11,9 +11,11 @@
     <converters:ShouldDisplayValueConverter x:Key="ShouldDisplayValueConverter" />
     <converters:PascalToPhraseConverter x:Key="PascalToPhraseConverter" />
     <converters:RpcStatusStringConverter x:Key="RpcStatusStringConverter" />
+    <converters:NetworkColorConverter x:Key="NetworkColorConverter"/>
+    <converters:NetworkStringConverter x:Key="NetworkStringConverter"/>
```

Comment:
This should be removed. reason https://github.com/zkSNACKs/WalletWasabi/pull/4044/commits/2a059b3123fcfc5f7ab8fe5038b7415fc2ea36b5
```suggestion
```